build/Microsoft.DotNet.Cli.Publish.targets

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(ToolsDir)/PublishContent.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)/publish/Microsoft.DotNet.Cli.Badge.targets" />
+
+  <UsingTask TaskName="UploadToAzure" AssemblyFile="$(BuildToolsTaskDir)/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+  <UsingTask TaskName="FinalizeBuildTask" AssemblyFile="$(CLIBuildDll)" />
+  <UsingTask TaskName="SetBlobPropertiesBasedOnFileTypeTask" AssemblyFile="$(CLIBuildDll)" />
+
+  <!-- PUBLISH_TO_AZURE_BLOB env variable set by CI -->
+  <Target Name="Publish"
+          Condition=" '$(PUBLISH_TO_AZURE_BLOB)' != '' "
+          DependsOnTargets="Prepare;
+                            Package;
+                            PublishArtifacts;
+                            FinalizeBuild" />
+
+  <Target Name="PublishArtifacts" DependsOnTargets="SetupAzureBlobInformation;
+                                                    GenerateVersionBadge;
+                                                    GatherItemsForPattern;
+                                                    UploadToAzure;
+                                                    PublishDebFilesToDebianRepo;
+                                                    PublishCliVersionBadge" />
+
+  <Target Name="FinalizeBuild">
+    <FinalizeBuildTask AccountName="$(CloudDropAccountName)"
+                       AccountKey="$(CloudDropAccessToken)"
+                       NugetVersion="$(NugetVersion)"
+                       Channel="$(Channel)"
+                       CommitHash="$(CommitHash)"
+                       BranchName="$(BranchName)" />
+  </Target>
+
+  <Target Name="SetupAzureBlobInformation">
+    <PropertyGroup>
+      <Product>Sdk</Product>
+      <ContainerName>$(STORAGE_CONTAINER)</ContainerName>
+      <CloudDropAccessToken>$(STORAGE_KEY)</CloudDropAccessToken>
```

Comment:
No, because a task defined by build tools expects these properties set with these names. It is outside of our control.